languages/ruby/exercises/concept/booleans/.meta/design.md

```diff
@@ -0,0 +1,45 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of nil in Ruby.
+
+## Learning objectives
+
+- Know that `true` is a Ruby object used to represent logical `true`
+- Know that everything but `false` and `nil` is truthy
+- Know that `false` is a Ruby object used to represent logical `false`
+- Know that `false` is falsey
+- Know that `nil` is a Ruby object used to represent an "empty" or "default" value
+- Know that `nil` is falsey
+
+## Out of scope
+
+- `TrueClass`
+- `FalseClass`
+- `NilClass`
```

Comment:
I added TrueClass and FalseClass because it originally had NilClass, but I think these may be good discussion points for the after document.